src/group/GroupAll.js

```diff
@@ -12,20 +12,22 @@ export default class GroupAll {
   _reduceExpression = null
   _maxCacheSize = 5
   /***********   CONSTRUCTOR   ***************/
-  constructor(dataConnector, crossfilter) {
-    this._init(dataConnector, crossfilter)
+  constructor(dataConnector, crossfilter, cache) {
+    this._init(dataConnector, crossfilter, cache)
     this._addPublicAPI()
   }
-  _init(dataConnector, crossfilter) {
+  _init(dataConnector, crossfilter, cache) {
     this._dataConnector     = dataConnector
-    this._cache             = new ResultCache(this._dataConnector)
+    this._cache             = cache
```

Comment:
GroupAll doesn't need it's own cache? If so, why is it stored in the instance vs using accessor methods into wherever it's created?